services/meta/data.go

```diff
@@ -163,11 +163,21 @@ func (data *Data) CreateRetentionPolicy(database string, rpi *RetentionPolicyInf
 		if rp.ReplicaN != rpi.ReplicaN || rp.Duration != rpi.Duration || rp.ShardGroupDuration != rpi.ShardGroupDuration {
 			return ErrRetentionPolicyExists
 		}
+		// if they want to make it default, and it's not the default, it's not an identical command so it's an error
+		if makeDefault && di.DefaultRetentionPolicy != rpi.Name {
```

Comment:
With my fix it is now an error.  While we don't know why they would do this (although we can assume it was a mistake), we can't alter an RP via a create if it exists.  You can only alter WITH an alter.  The point of this code is so that if the create is issued with identical specifications, it's idempotent.  The intent is to allow them to run the same scripts over and over for create database or retention policy without it erring out when the specifications are identical.  If they change their script, then they will have to manually fix any changes to have it run in an idempotent manner again.